workshop/content/intermediate/templates/resource-dependencies/_index.md

```diff
@@ -0,0 +1,275 @@
+* Note that there are no dependencies between `SNSTopic` and `SecurityGroup` resources: this means that CloudFormation initiates the creation of both these resources in parallel.
+
+Let’s create a stack, and verify this is the expected behavior. Use the AWS CloudFormation Console to [create a stack](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/cfn-console-create-stack.html) using the `resource-dependency-with-intrinsic-functions.yaml` template:
+
+
+1. Navigate to the [AWS CloudFormation Console](https://console.aws.amazon.com/cloudformation/).
+2. From **Create stack**, choose **With new resources (standard)**.
+3. Choose the **Template is ready** option. From **Specify template**, choose **Upload a template file**. Upload the `resource-dependencies-with-intrinsic-functions.yaml` template, and choose **Next**.
+4. Enter a stack name. For example, `resource-dependencies-lab-ref-getatt`.
+5. In the **Parameters** section, provide a unique name for the SNS topic, an email address for SNS Topic subscription, and a name for the security group; when ready, choose **Next**.
+6. Choose to accept default values on the **Configure stack options** page; scroll to the bottom of the page, and choose **Next**.
+7. In the review page, scroll to the bottom and choose **Create stack**.
+
+
+Once the stack is created, your stack events should look like in the following example:
+
+![resource-dependencies-lab-ref-getatt.png](resource-dependencies/resource-dependencies-lab-ref-getatt.png)
+
+Let’s review stack events for your `resource-dependencies-lab-ref-getatt` stack. Note the creation of `SNSTopic` and `SecurityGroup` resources started in parallel, as both resources had no dependencies between each other. Also, note the `SecurityGroupIngress` resource creation started only after the `SecurityGroup` resource was in the `CREATE_COMPLETE` status, and the `SNSTopicSubscription` creation started after the `SNSTopic` resource was successfully created.
+
+When you will delete your stack, CloudFormation follows creation order in reverse: in this case, for example, `SNSTopicSubscription` and `SecurityGroupIngress` resources will be deleted first, followed by `SecurityGroup` and `SNSTopic`.
+
+{{% notice note %}}
+You should have received an email, sent to the email address you provided, for you to confirm the subscription to the SNS topic you created. Choose to follow the subscription link in the subscription confirmation email you received for the topic you created, to subscribe to your topic: otherwise, when you delete the stack, the subscription will be left in pending state and will not be deleted from your account. For more information, see [Deleting an Amazon SNS subscription and topic](https://docs.aws.amazon.com/sns/latest/dg/sns-delete-subscription-topic.html).
```

Comment:
```suggestion
You should have received an email, sent to the email address you provided, for you to confirm the subscription to the Amazon SNS topic you created. Choose to follow the subscription link in the subscription confirmation email you received for the topic you created, to subscribe to your topic: otherwise, when you delete the stack, the subscription will be left in pending state and will not be deleted from your account. For more information, see [Deleting an Amazon SNS subscription and topic](https://docs.aws.amazon.com/sns/latest/dg/sns-delete-subscription-topic.html).
```